Lombiq.HelpfulLibraries.OrchardCore/TagHelpers/EditorFieldSetTagHelper.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using System;
+using System.Threading.Tasks;
+
+namespace Lombiq.HelpfulLibraries.OrchardCore.TagHelpers;
+
+[HtmlTargetElement("fieldset", Attributes = "asp-for")]
+public class EditorFieldSetTagHelper : TagHelper
+{
+    private readonly IHtmlGenerator _htmlGenerator;
+
+    [HtmlAttributeNotBound]
+    [ViewContext]
+    public ViewContext ViewContext { get; set; }
+
+    [HtmlAttributeName("asp-for")]
+    public ModelExpression For { get; set; }
+
+    [HtmlAttributeName("label")]
+    public LocalizedHtmlString Label { get; set; }
+
+    [HtmlAttributeName("hint")]
+    public LocalizedHtmlString Hint { get; set; }
+
+    [HtmlAttributeName("type")]
+    public string InputType { get; set; } = "text";
+
+    public EditorFieldSetTagHelper(IHtmlGenerator htmlGenerator) =>
+        _htmlGenerator = htmlGenerator;
+
+    public override Task ProcessAsync(TagHelperContext context, TagHelperOutput output)
+    {
+        if (output.Attributes.TryGetAttribute("class", out var classAttribute))
+        {
+            var newValue = $"{classAttribute.Value} form-group";
+            output.Attributes.Remove(classAttribute);
+            output.Attributes.Add("class", newValue);
+        }
+        else
+        {
+            output.Attributes.Add("class", "form-group mb-3");
```

Comment:
Since this helper is OC specific, let's add another default (?) style - `col-xl-6`. The current layout with the input fields spanning the whole screen width doesn't look that great, and I found at least one other settings page that uses it:

![image](https://github.com/Lombiq/Helpful-Libraries/assets/164360/789db5a8-543a-470f-9bf9-f0e96b581075)

```suggestion
            output.Attributes.Add("class", "form-group mb-3 col-xl-6");
```

Reply:
I haven't see this class in our code. But sure, why not?